src/WebJobs.Script/OutOfProc/Rpc/LanguageWorkerProcessFactory.cs

```diff
@@ -42,7 +42,7 @@ public ILanguageWorkerProcess CreateLanguageWorkerProcess(string workerId, strin
         {
             WorkerConfig workerConfig = _workerConfigs.Where(c => c.Language.Equals(runtime, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
             ILogger workerProcessLogger = _loggerFactory.CreateLogger($"Worker.LanguageWorkerProcess.{runtime}.{workerId}");
-            return new LanguageWorkerProcess(runtime, workerId, scriptRootPath, _rpcServer.Uri, workerConfig.Arguments, _eventManager, _workerProcessFactory, _processRegistry, workerProcessLogger, _consoleLogSource);
```

Comment:
Should the type be renamed since it's now creating an instance of an `RpcWorkerProcess`?